src/main/java/net/sf/jabref/model/entry/InternalBibtexFields.java

```diff
@@ -184,7 +184,8 @@ private InternalBibtexFields() {
         add(new BibtexSingleField("eid", true, BibtexSingleField.SMALL_W));
 
         dummy = new BibtexSingleField("date", true);
-        dummy.setPrivate();
+        dummy.setExtras(EnumSet.of(FieldProperties.DATE));
+        dummy.setPrivate(); // TODO: Why private?
```

Comment:
I currently have no clear picture about private fields. This is the context of https://github.com/JabRef/jabref/issues/574 :smile: